generators/python-v2/ast/src/Class.ts

```diff
@@ -35,6 +35,10 @@ export class Class extends AstNode {
         this.extends_.forEach((parentClassReference) => {
             this.inheritReferences(parentClassReference);
         });
+
+        this.decorators.forEach((decorator) => {
+            this.inheritReferences(decorator);
```

Comment:
[q] the decorator could be a function with arguments. Does this catch those parameters?